Tests ES6 harmony scope sections.

Set timer for test function.
Script execution paused.

Scope variables sidebar pane:
Block
    block2: "catch(e) {...}"
    const2: 2
Catch block
    e: Error: An exception
    at innerFunction (...)
    at testFunction (...)
Block
    block1: "block {...}"
    const1: 1
Local
    this: undefined
    innerFunctionBlockVar: 2116
    innerFunctionLocalVar: 2016
    negInf: -Infinity
    negZero: -0
    x: 2014
Block
    makeClosureDeeperBlockVar: "block.deep.TextParam"
Closure (window.makeClosure)
    makeClosureBlockVar: "block.TextParam"
    makeClosureDeeperLocalVar: "local.deep.TextParam"
    makeClosureLocalVar: "local.TextParam"
    n: "TextParam"
Script
    globalScriptConst: 42
    globalScriptLet: 41
WindowGlobal
    <section collapsed>
Script execution resumed.

